pom.xml

```diff
@@ -548,7 +549,7 @@
           <plugin>
             <groupId>org.codehaus.mojo</groupId>
             <artifactId>flatten-maven-plugin</artifactId>
-            <!--<version>1.2.2</version>-->
+            <version>1.2.2</version>
```

Comment:
why this change ?

Reply:
maven will throw many warning info at the begin of building if we not set it.